sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Processor/PartitionLoadBalancer.cs

```diff
@@ -420,12 +420,23 @@ public virtual async Task RelinquishOwnershipAsync(CancellationToken cancellatio
                     {
```

Comment:
Are the if and the else if the same? Maybe they could be condensed into one?

Reply:
They are close, but use different collections of ownership.   It could be refactored to move the logic into a function and then call it with the different collections, but I find it worth the efforts since it's simple logic that repeats.   I'm going to let that stand for now.   At some point, there's a good amount of the load balancer that we should look at for refactoring for efficiency and code cleanup.

Reply:
Ah I see, makes sense!